export-to-png.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
```

Comment:
@mojavelinux On my machine (Ubuntu 18.04 LTS), the path should be `#!/bin/bash` otherwise I get a `zsh: ./export-to-png.sh: bad interpreter: /usr/bin/bash`

Reply:
Thanks for the heads up. I'll update it.